app/events/applications/application_created_event.rb

```diff
@@ -8,13 +10,11 @@ def self.create(application, user)
       application: application,
       account:     account = application.user_account,
       provider:    provider,
-      # this really can't be application.service as it would break:
-      # $ rspec ./spec/acceptance/api/application_spec.rb -e 'Cinstance application json format GET /admin/api/applications/find.:format with app id Get Application'
```

Comment:
I can't find this test, and I can't see how `GET ../find` endpoint would trigger `ApplicationCreatedEvent`... So I am assuming there was some confusion here.